Makefile

```diff
@@ -258,6 +259,14 @@ test-go:
 test-python:
 	@./build/test.sh python
 
+test-e2e-existing-clusters:
+	@eval $$(python3 ./manager/cluster_config_env.py ./dev/config/cluster-aws.yaml) && CORTEX_CLI_PATH=./bin/cortex ./build/test.sh apis -p aws -e "$$CORTEX_CLUSTER_NAME-aws"
+	@eval $$(python3 ./manager/cluster_config_env.py ./dev/config/cluster-gcp.yaml) && CORTEX_CLI_PATH=./bin/cortex ./build/test.sh apis -p gcp -e "$$CORTEX_CLUSTER_NAME-gcp"
```

Comment:
would be useful to run these in parallel. Otherwise, you can just run a single command